### ISSUE TITLE
Patch for /trunk/AndorsTrail/res/values/strings.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#199]
```
English proofread.

One typo corrected. ;)
```

Original issue reported on code.google.com by `medic....@gmail.com` on 8 Jun 2011 at 11:16

Attachments:
- [strings.xml.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-199/comment-0/strings.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks again. Committed in r101.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2011 at 9:12
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.9**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:53
- Changed state: **Done**
